docker-images/Dockerfile.oni-runtime-debug

```diff
@@ -0,0 +1,28 @@
+ARG GO_VERSION=1.14.2
+
+FROM golang:${GO_VERSION}-buster as downloader
+
+## Fetch the proof parameters.
+## 1. Install the paramfetch binary first, so it can be cached over builds.
+## 2. Then copy over the parameters (which could change).
+## 3. Trigger the download.
+## Output will be in /var/tmp/filecoin-proof-parameters.
+
+RUN go get github.com/filecoin-project/go-paramfetch/paramfetch
+COPY /proof-parameters.json /
+RUN paramfetch 2048 /proof-parameters.json
+
+## for debug purposes
+RUN apt update && apt install -y mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl && git clone https://github.com/filecoin-project/lotus.git && cd lotus/ && git checkout next && make clean && make all && make install
```

Comment:
Depending on how often these tools (`lotus` and `lotus-storage-miner`) change, we might want to move these utilities directly in the `testplan` Dockerfile. For now I have had some success by bundling them with the `runtime` image, but it is rather annoying when they change, as I have to recompile, tag, push images.